spec/pods.md

```diff
@@ -172,6 +173,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
 * **volumes** (list of objects, optional) list of volumes which will be mounted into each application's filesystem
     * **name** (string, required) descriptive label for the volume (restricted to the [AC Name](types.md#ac-name-type) formatting), used as an index by the `mounts` objects (above).
     * **readOnly** (boolean, optional, defaults to "false" if unsupplied) whether or not the volume will be mounted read only.
+    * **recursive** (boolean, optional) whether or not the volume will be mounted recursively. This only applies to volumes of type **host**. Executors MUST ignore **recursive** on **empty** volumes. When **recursive** is not specified, the executor SHOULD default to recursive. However, executors in specific run environment might prefer to default to non-recursive.
```

Comment:
sorry, couple of things I should have mentioned on first pass:
- s/type/kind/
- can you move this after the **kind** entry? (i.e. before/after "source") - you can adopt similar language to mode/uid - "boolean, optional, only interpreted if **kind** is "host""
- can you link to a reference to what recursive means?